packages/zoe/test/unitTests/contracts/attestation/test-attestation.js

```diff
@@ -0,0 +1,278 @@
+
+  const address1 = 'address1';
+
+  /** @type {AttMaker} */
+  const attMaker = await E(creatorFacet).getAttMaker(address1);
+
+  // Try to make a lien greater than the total
+  const largeAmount = AmountMath.make(uBrand, 1000n);
+  const longExpiration = 100n;
+  const shortExpiration = 15n;
+  await t.throwsAsync(
+    () => E(attMaker).makeAttestations(largeAmount, shortExpiration),
+    {
+      message:
+        'Only {"brand":"[Alleged: BLD brand]","value":"[500n]"} was unliened, but an attestation was attempted for {"brand":"[Alleged: BLD brand]","value":"[1000n]"}',
+    },
+  );
+
+  // Try to make a lien with an already expired expiration
+  const amount50 = AmountMath.make(uBrand, 50n);
+  const expiredExpiration = 1n;
+  await t.throwsAsync(
+    () => E(attMaker).makeAttestations(amount50, expiredExpiration),
+    { message: 'Expiration "[1n]" must be after the current time "[10n]"' },
+  );
+
+  // check that no lien was successful so far
+  const liened = await E(creatorFacet).getLiened(
+    harden([address1]),
+    currentTime,
+  );
+  t.deepEqual(liened, [AmountMath.makeEmpty(uBrand)]);
+
+  // Make a normal lien
+  const { returnable: returnable1, expiring: expiring1 } = await E(
+    attMaker,
+  ).makeAttestations(amount50, shortExpiration);
+
+  t.deepEqual(
+    await E(issuers.returnable).getAmountOf(returnable1),
+    AmountMath.make(brands.returnable, [
+      {
+        address: 'address1',
+        amountLiened: amount50,
+      },
+    ]),
+  );
+  const expiring1Amount = await E(issuers.expiring).getAmountOf(expiring1);
+  t.deepEqual(
+    expiring1Amount,
+    AmountMath.make(brands.expiring, [
+      {
+        address: 'address1',
+        amountLiened: amount50,
+        expiration: shortExpiration,
+        handle: expiring1Amount.value[0].handle,
+      },
+    ]),
+  );
+
+  const liened2 = await E(creatorFacet).getLiened(
+    harden([address1]),
+    currentTime,
+  );
+  t.deepEqual(liened2, [amount50]);
+
+  // Make another normal lien
+
+  const amount25 = AmountMath.make(uBrand, 25n);
+  const { returnable: returnable2, expiring: expiring2 } = await E(
+    attMaker,
+  ).makeAttestations(amount25, longExpiration);
+
+  t.deepEqual(
+    await E(issuers.returnable).getAmountOf(returnable2),
+    AmountMath.make(brands.returnable, [
+      {
+        address: 'address1',
+        amountLiened: amount25,
+      },
+    ]),
+  );
+  const expiring2Amount = await E(issuers.expiring).getAmountOf(expiring2);
+  t.deepEqual(
+    expiring2Amount,
+    AmountMath.make(brands.expiring, [
+      {
+        address: 'address1',
+        amountLiened: amount25,
+        expiration: longExpiration,
+        handle: expiring2Amount.value[0].handle,
+      },
+    ]),
+  );
+  t.not(expiring1Amount.value[0].handle, expiring2Amount.value[0].handle);
+
+  const liened3 = await E(creatorFacet).getLiened(
+    harden([address1]),
+    currentTime,
+  );
+  t.deepEqual(liened3, [AmountMath.add(amount50, amount25)]);
```

Comment:
Should this be `t.assert(AmountMath.isEqual(liened3, [AmountMath.add(amount50, amount25)])`?